Makefile

```diff
@@ -142,6 +150,89 @@ calico/felix: bin/calico-felix
 	cp bin/calico-felix docker-image/bin/
 	docker build -t calico/felix docker-image
 
+# Targets for Felix testing with the k8s backend and a k8s API server,
+# with k8s model resources being injected by a separate test client.
+GET_CONTAINER_IP := docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'
+K8S_VERSION=1.5.3
+FELIX_K8S=felix-k8s
+.PHONY: k8s-fv-test run-k8s-apiserver stop-k8s-apiserver run-etcd stop-etcd
+k8s-fv-test: calico/felix run-k8s-apiserver k8sfv/k8sfv.test
+	@-docker rm -f $(FELIX_K8S)
+	sleep 1
+	K8S_IP=`$(GET_CONTAINER_IP) calico-k8s-apiserver` && \
+	docker run --detach --privileged --name=$(FELIX_K8S) \
```

Comment:
No, because:

- Sometimes I screw some other piece of config up, that causes this container to exit shortly after start up, and when that happens I want to be able to look at its logs.

- We have another mechanism to avoid leaving lots of zombie containers around: using a specific name, and doing `docker rm` on that name just before starting a new one.